src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -1617,6 +1625,8 @@ private static void ReadStack(SparseReader reader, object parent)
             Debug.Assert(context.CurrentThreadFlowLocation == null);
             context.CurrentThreadFlowLocation = new ThreadFlowLocation
             {
+                Location = context.Signature.Location,
```

Comment:
>Location [](start = 16, length = 8)

After all the fuss and bother, this one-liner is the fix for #1530. The rest was just making sure the fully qualified logical name was in the right form.

Reply:
i don't like what we've done to FQN, we've elided the information that qualifies one method overload over the other. this isn't heplful and will compromise result matching.

---
In reply to: [293025496](https://github.com/microsoft/sarif-sdk/pull/1531#discussion_r293025496) [](ancestors = 293025496)

Reply:
Same as the other two comments -- I will restore the signature to the FQN.

---
In reply to: [293604799](https://github.com/microsoft/sarif-sdk/pull/1531#discussion_r293604799) [](ancestors = 293604799,293025496)